.gitmodules

```diff
@@ -28,8 +28,8 @@
 	shallow = true
 [submodule "src/llvm-project"]
 	path = src/llvm-project
-	url = https://github.com/rust-lang/llvm-project.git
-	branch = rustc/19.1-2024-12-03
+	url = https://github.com/nikic/llvm-project.git
+	branch = rust-llvm-20
```

Comment:
Is this change intended?

Reply:
Nope, that was a mistake I made when extracting these changes. Thanks for catching it!